main.js

```diff
@@ -253,7 +253,7 @@ class HmIpCloudAccesspointAdapter extends Adapter {
                                 return;
                         }
                         const pin = await this.getStateAsync(`devices.${o.native.id}.channels.${o.native.channel}.pin`);
-                        this.log.info(`Call setLockState for ${state.val} with PIN ${pin ? pin.val : ''}`);
+                        this.log.info(`Call setLockState for ${state.val}`);
```

Comment:
what do you think about this:

```suggestion
                        this.log.info(`Call setLockState for ${state.val} ${pin ? 'with' : 'without'} PIN`);
```

Reply:
Ok I added that and a changelog entry. Thank you!